### PR TITLE
Dev tweak test

### PR DESCRIPTION
**Pull request by privateOmega**
Extracted checkIfAbsoluteURL() to a helper file.

## Description
Created a helper file at test/helpers/index.js.
Extracted checkIfAbsoluteURL() to it.
Referenced helper file at test file.

## Related Issue
https://github.com/DAVFoundation/missioncontrol/issues/129

## How Has This Been Tested?
npm test
Passed all the 5 tests.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@turf/turf": "^4.7.3",
     "bluebird": "^3.5.1",
     "express": "^4.16.2",
+    "pinkie": "^2.0.4",
```
Comment (TalAter) [inline package.json]: Can you remove this change? It has nothing to do with this PR.
Same goes for any changes to `package-lock.json`

**Comment by TalAter** [comment]
Just a small comment

**Comment by TalAter** [comment]
That's an excellent pull request! 👍 :octocat: 

Thank you so much @privateOmega 

I hope to see more contributions from you!

**Comment by privateOmega** [comment]
@TalAter Happy to help in whatever way I can.

**Pull request merged**